deep_speech_2/tests/test_error_rate.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import unittest
+import sys
+sys.path.append('..')
+import error_rate
+
+
+class TestParse(unittest.TestCase):
+    def test_wer(self):
+        ref = 'i UM the PHONE IS i LEFT THE portable PHONE UPSTAIRS last night'
+        hyp = 'i GOT IT TO the FULLEST i LOVE TO portable FROM OF STORES last night'
+        word_error_rate = error_rate.wer(ref, hyp)
```

Comment:
Could we add more test cases?
e.g. 
self.assertTrue(error_rate.wer(ref, ref) == 0)
test if ValueError is raised if len(ref) == 0.